### ISSUE TITLE
Can we close CMD window after injecting?

### ISSUE THREAD

**Issue by KevinTyrrell** [namreeb/nampower#23]
After running the loader which launches World of Warcraft, are we free to close the CMD instance that ran `loader.exe`? For example  I have the following `.bat` script.

```bat
loader.exe -p WoW.exe
exit
```

**Comment by namreeb** [comment]
Should be fine.  You could also create a desktop shortcut (.lnk) file to do it instead.

**Comment by KevinTyrrell** [comment]
Quick question, are oddities with animations normal using this? On Everlook no 'start casting' animation occurs as I repeated spam cast an ability (e.g. Smite). It's only when the ability finishes casting that the game realizes I've casted and therefore does the animation (e.g. Undead putting his hand forward). Seems like the castbar also falls out of sync, though none of this really matters for dpsing in raids on a class like Mage or Warlock.
